web/src/stories/FrozenTableStory.js

```diff
@@ -0,0 +1,157 @@
+                  <td className="mono right-align nowrap " headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q3">--</td>
+                  <td className="mono right-align nowrap " headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q4">--</td>
+                  <td className="bold mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-subtotal">--</td>
+                  <td className="bold mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-total2 quarterly-budget-summary-hitAndHie-total">--</td>
+                </tr>
+                <tr className="bold"><td headers="quarterly-budget-summary-hitAndHie-null1 quarterly-budget-summary-hitAndHie-null2">Total Enhanced FFP</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2019 quarterly-budget-summary-hitAndHie-fy-2019-q1">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2019 quarterly-budget-summary-hitAndHie-fy-2019-q2">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2019 quarterly-budget-summary-hitAndHie-fy-2019-q3">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2019 quarterly-budget-summary-hitAndHie-fy-2019-q4">--</td>
+                  <td className="bold mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2019 quarterly-budget-summary-hitAndHie-fy-2019-subtotal">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q1">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q2">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q3">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q4">--</td>
+                  <td className="bold mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-subtotal">--</td>
+                  <td className="bold mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-total2 quarterly-budget-summary-hitAndHie-total">--</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </RenderViewbox>
+      <h2>The basic structure to create this is as follows:</h2>
+      <pre>
+        <code>
+          <div className="table-frozen-wrapper">
```

Comment:
If you wrap the contents of this `code` element in ``{` `}`` then it'll render as as string instead of rendering the HTML.
```suggestion
          {`<div className="table-frozen-wrapper">
```

Reply:
Oh! Excellent, thank you!